addOns/commonlib/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 ### Added
 - Constants related to authentication.
+- Added Alert Tags to be used to indicate (ex: via Example Alerts) alerts (rules) which have been mapped to the PCI-DSS or HIPAA standard(s). 
```

Comment:
`- Added Alert Tags for PCI DSS and HIPAA standards.`

Reply:
`PCI DSS` should be updated everywhere.

Reply:
Done.
I'd rather the tag had a hyphen, but I also don't have a good argument why it needs one :wink:

Reply:
If the problem is the space add an underscore like other tags have.

Reply:
Naw it's fine.